Author: Marcus Blumhagen <marcus.blumhagen@web.de>
Comment: Fixes LP: #482408 where the stream volume would erroneously be set to max for each new mplayer invocation

Index: mplayer-1.0~rc3+svn20090426/libao2/ao_pulse.c
===================================================================
--- mplayer-1.0~rc3+svn20090426.orig/libao2/ao_pulse.c	2009-11-18 19:31:12.000000000 -0500
+++ mplayer-1.0~rc3+svn20090426/libao2/ao_pulse.c	2009-11-18 19:31:54.000000000 -0500
@@ -221,7 +221,7 @@
     pa_stream_set_write_callback(stream, stream_request_cb, NULL);
     pa_stream_set_latency_update_callback(stream, stream_latency_update_cb, NULL);
 
-    if (pa_stream_connect_playback(stream, sink, NULL, PA_STREAM_INTERPOLATE_TIMING|PA_STREAM_AUTO_TIMING_UPDATE, &volume, NULL) < 0)
+    if (pa_stream_connect_playback(stream, sink, NULL, PA_STREAM_INTERPOLATE_TIMING|PA_STREAM_AUTO_TIMING_UPDATE, NULL, NULL) < 0)
         goto unlock_and_fail;
 
     /* Wait until the stream is ready */
